oaknational/Oak-Web-Application
# Fix/docx pathways

## Description

Added `pathway` to docx document generation. 

## How to test

1. Go to {owa_deployment_url}/teachers/curriculum/docx-poc/select
2. Generate a document for a subject with pathways
3. Document should be generated correctly
